server/server.go

```diff
@@ -49,7 +54,10 @@ func getSupportedLanguage(w http.ResponseWriter, r *http.Request) {
 
 }
 
-// upload parses the client request and uploads the file.
+// upload parses the client request, reads and stores the form data.
+// It uploads the assignment tarball, stores the commands to compile and run.
```

Comment:
incorrect comment. Does this method upload files?

Reply:
Agreed. Removed the upload part from the comment